CMakeLists.txt

```diff
@@ -94,5 +94,5 @@ include(AzureVersion)
 add_subdirectory(sdk/core)
 add_subdirectory(sdk/identity)
 add_subdirectory(sdk/keyvault)
-add_subdirectory(sdk/storage)
+#add_subdirectory(sdk/storage)
```

Comment:
Is this intentional?

Reply:
Yes, moving namespaces breaks storage all around.
It will be disabled until @Jinming-Hu  fix the renaming

Reply:
Can we fix the rename in Storage and get Jinming's approval?.

Reply:
nop, because mostly the changes are the the auto generated rest clients and jinming would just remove my commit and re-apply to his auto-gen tools